src/Microsoft.Performance.SDK.Runtime.NetCoreApp/Plugins/PluginsLoader.cs

```diff
@@ -195,16 +198,19 @@ public bool TryLoadPlugin(string directory, out ErrorInfo error)
         }
 
         /// <summary>
-        ///     Asynchronous version of <see cref="TryLoadPlugin(string)"/>
+        ///     Asynchronous version of <see cref="TryLoadPlugin(string, out ErrorInfo)"/>
         /// </summary>
         /// <exception cref="ObjectDisposedException">
         ///     This instance is disposed.
         /// </exception>
-        public async Task<(bool, IDictionary<string, ErrorInfo>)> TryLoadPluginAsync(IEnumerable<string> directories)
+        /// <exception cref="ArgumentNullException">
+        ///     <paramref name="directory"/> is null.
+        /// </exception>
+        public async Task<(bool, IDictionary<string, ErrorInfo>)> TryLoadPluginAsync(IEnumerable<string> directory)
```

Comment:
Why is `directories` changed to singular?

nit: Add <param>

Reply:
I initially thought it was just a typo, but then realized the signature is wrong. #222 

I'll revert this for this PR

Reply:
Sorry I meant

nit: Add the 'param' doc string for directories

It didn't show up below

Reply:
Is it OK to do when I fix #222 in a separate PR? If I added a param docstring now it'll be out of date once I fix it anyway

Reply:
Yeah sounds good!